mappings/net/minecraft/client/realms/task/CreateWorldTask.mapping

```diff
@@ -1 +1,10 @@
 CLASS net/minecraft/unmapped/C_hspxlahv net/minecraft/client/realms/task/CreateWorldTask
+	FIELD f_dhvsbwdt motd Ljava/lang/String;
+	FIELD f_drnsitdd name Ljava/lang/String;
+	FIELD f_kiapcmwd worldId J
+	FIELD f_lprvtdmf MCO_CREATE_WORLD_WAIT Lnet/minecraft/unmapped/C_rdaqiwdt;
```

Comment:
instead of strictly naming it based on the key, I'd just call this `TITLE`
```suggestion
	FIELD f_lprvtdmf TITLE Lnet/minecraft/unmapped/C_rdaqiwdt;
```